server/src/main/java/umm3601/Server.java

```diff
@@ -41,22 +42,19 @@ public static void main(String[] args) {
 
     Javalin server = Javalin.create().start(4567);
 
-    // Simple example route
-    server.get("hello", ctx -> ctx.result("Hello World"));
-
     // Utility routes
-    server.get("api", ctx -> ctx.result(appName));
+    server.get("/api", ctx -> ctx.result(appName));
 
     // Get specific user
-    server.get("api/users/:id", userController::getUser);
+    server.get("/api/users/:id", userController::getUser);
```

Comment:
The order of these paths is kind random and probably reflects history more than anything else. I might re-arrange them and add that to the PR.